railties/lib/rails/health_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Rails::HealthController < ActionController::Base # :nodoc:
+  rescue_from(Exception) { render_down }
+
+  def show
+    render_up
+  end
+
+  private
+    def render_up
+      render html: html_status(color: "green")
+    end
+
+    def render_down
+      render html: html_status(color: "red"), status: 500
```

Comment:
Wouldn't `503 Service Unavailable` be a more appropriate HTTP code in this case?

Reply:
Yes.

Reply:
Here is a PR to address this: https://github.com/rails/rails/pull/47061

Reply:
💯